scripts/install.sh

```diff
@@ -91,4 +97,3 @@ bluemix_auth
 cluster_setup
 initial_setup
 getting_ip_port
-cluster_setup
```

Comment:
cluster_setup is for cleaning all the previous deployment, we still called in line 97. Previously I put it at the end to clear all the deployment because I do not want other people to abuse the OpenWhisk on Travis.